pkg/controller/servicebinding/reconciler.go

```diff
@@ -78,10 +78,10 @@ func (r *reconciler) getServiceBinding(
 	return sbr, nil
 }
 
-func servicesOrEmptyList(maybeServices *[]v1alpha1.Service) []v1alpha1.Service {
+func servicesOrEmptyList(maybeServices []v1alpha1.Service) []v1alpha1.Service {
```

Comment:
this function is now pointless and can be removed fully, because `maybeServices` is not pointer anymore, this cannot be `nil`.

Reply:
Removed.